frontend/components/Home/Dashboard/Dashboard.tsx

```diff
@@ -62,6 +60,40 @@ const Dashboard = () => {
     const [logOpen, setLogOpen] = useState(false);
     const [logModal, setLogModal] = useState(false);
 
+    const [messageDisp, setMessageDisp] = useState(false);
+   
+    const [tipCard, setTipCard] = useState({title: "", description: "", student: false});
+
+    var tipsArr = tips;
+
+    useEffect(() => {
```

Comment:
Since there's a not insubstantial logic associated with the tip card, it might be best to spool it out into it's own component.